api/v1alpha1/workspace_validation.go

```diff
@@ -299,16 +300,23 @@ func (r *DataDestination) validateUpdate(old *DataDestination) (errs *apis.Field
 	return errs
 }
 
-func (r *ResourceSpec) validateCreate(inference InferenceSpec) (errs *apis.FieldError) {
+func (r *ResourceSpec) validateCreateWithTuning(tuning *TuningSpec) (errs *apis.FieldError) {
+	if tuning != nil && *r.Count > 1 {
+		errs = errs.Also(apis.ErrInvalidValue("Tuning does not currently support multinode configurations. Please set the node count to 1. Future support with DeepSpeed will allow this.", "count"))
+	}
+	return errs
+}
+
+func (r *ResourceSpec) validateCreateWithInference(inference *InferenceSpec) (errs *apis.FieldError) {
 	var presetName string
-	if inference.Preset != nil {
+	if inference != nil && inference.Preset != nil {
```

Comment:
Caller side already does the check, inference != nil  is not needed.

Reply:
Fixed